src/js/index.js

```diff
@@ -0,0 +1,55 @@
+import Expression from "./Expression.js"
+
+
+const selectElements = (query) => {
+    return document.querySelectorAll(query);
+}
+
+const selectElement = (query) => {
+    return document.querySelector(query);
+}
+
+const setClickEvts = (elements, callback) => {
+    elements.forEach(element => {
+        element.addEventListener('click', callback())
+    });
+}
+
+window.onload = ()=>{
+    const expression = new Expression();
+
+    const total = selectElement('#total');
+    const setTotal = (text) =>{
+        total.innerText=text;
+    }
+
+    const getOperand = () =>{
+        return (event) => {
+            expression.getOperand(event.target.innerText);
+            setTotal()
+        }
+    }
+
+    const getOperator = () => {
+        return (event) => {
+            expression.getOperator(event.target.innerText);
+            const result = expression.getResult();
+            if (result !== null) {
+                document.querySelector('#total').innerHTML = result;
+                setTotal(result)
```

Comment:
어차피 null 은 falsey 값이기 때문에 아래처럼 간단하게도 쓸 수 있답니당!!
```suggestion
                if(!result) return;
                document.querySelector('#total').innerHTML = result;
                setTotal(result)
```